content/webinar/_index.md

```diff
@@ -2,4 +2,5 @@
 private: true
 title: Webinars
 meta_desc: Learn best practices and innovating techniques when interacting with cloud resources.
+redirect_to: /webinars
```

Comment:
Perhaps cleaner: delete this file, and update `content/webinars/_index.md` to have the following front matter property:

```
aliases:
- /webinar/
```

Hugo will then generate a /webinar/ page that has the meta-redirect, and when we publish the site, we'll convert that to a proper 301 redirect.